### PR TITLE
Team discovery settings

### PR DESCRIPTION
**Pull request by ThomasGaubert**

### CODE REVIEW

**Comment by Lauragra** [comment]
@nkramer , should this one be set to ready to merge?

File: concepts/changelog.md [hunk 1]
```diff
@@ -66,6 +67,7 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | :-------------- | :------------ | :--------------------------------------- |
 |Addition |v1.0 | Added support for application permissions to the [installedApps](/graph/api/resources/teamsappinstallation?view=graph-rest-1.0) resource.|
 |Addition |v1.0| Added the **email** and **webUrl** properties to [channel](/graph/api/resources/channel?view=graph-rest-1.0).|
+| Addition | beta | Added the [teamDiscoverySettings](/api-reference/beta/resources/teamdiscoverysettings.md) resource and associated methods. |
```
Comment (Lauragra) [inline concepts/changelog.md]: Does this add the entry to the Intune table? I think you need to add a new H3 heading under April for Microsoft Teams API and add a table there.

File: api-reference/beta/resources/teamdiscoverysettings.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+localization_priority: Normal
+author: "nkramer"
+ms.prod: "microsoft-teams"
+---
+
+# teamDiscoverySettings resource type
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Settings to configure [team](team.md) discoverability by others. A team's discovery settings may only be modified if the team is private.
```
Comment (Lauragra) [inline api-reference/beta/resources/teamdiscoverysettings.md]: Would it be more clear to revise as "Provides settings to enable others to configure team discoverability."? The second sentence also might be more readable if recast as active voice: "You can only modify discovery settings for private teams."

File: concepts/changelog.md (hunk 1 above)
Comment (ThomasGaubert) [reply concepts/changelog.md]: The diff snippet cuts off the Microsoft Teams API header. The code is currently:

```markdown
### Microsoft Teams APIs

| **Change type** | **Version**   | **Description**                          |
| :-------------- | :------------ | :--------------------------------------- |
|Addition |v1.0 | Added support for application permissions to the [installedApps](/graph/api/resources/teamsappinstallation?view=graph-rest-1.0) resource.|
|Addition |v1.0| Added the **email** and **webUrl** properties to [channel](/graph/api/resources/channel?view=graph-rest-1.0).|
| Addition | beta | Added the [teamDiscoverySettings](/api-reference/beta/resources/teamdiscoverysettings.md) resource and associated methods. |
```

**Pull request merged**